src/MySqlConnector/MySqlClient/MySqlErrorCode.cs

```diff
@@ -0,0 +1,18 @@
+namespace MySql.Data.MySqlClient
+{
+	/// <summary>
+	/// MySQL Server error codes. Taken from <a href="https://dev.mysql.com/doc/refman/5.7/en/error-messages-server.html">Server Error Codes and Messages</a>.
+	/// </summary>
+	public enum MySqlErrorCode
```

Comment:
Could do `public static class MySqlErrorCodes` with `public const int` fields to avoid casting.